client/packages/common/src/authentication/api/hooks/useAuthApi.ts

```diff
@@ -6,14 +6,16 @@ import { getSdk } from '../operations.generated';
 export const useAuthApi = () => {
   const { client } = useGql();
   const t = useTranslation();
-  const queries = getAuthQueries(getSdk(client), t);
+  const sdk = getSdk(client);
+  const queries = getAuthQueries(sdk, t);
 
   const keys = {
     me: (token: string) => ['me', token] as const,
     isCentralServer: ['isCentralServer'] as const,
     refresh: (token: string) => ['refresh', token] as const,
     userSync: () => ['userSync'] as const,
+    preferences: () => ['preferences'] as const,
   };
 
-  return { ...queries, keys };
+  return { ...queries, sdk, keys };
```

Comment:
exporting sdk here to align more with the api hook pattern we are moving towards